example/ipsec_api/odp_ipsec_fwd_db.h

```diff
@@ -0,0 +1 @@
+../ipsec/odp_ipsec_fwd_db.h
```

Comment:
Similar issues here. Missing newline?
```
WARNING: adding a line without newline at end of file
#1759: FILE: example/ipsec_api/odp_ipsec_fwd_db.h:1:
+../ipsec/odp_ipsec_fwd_db.h
```

Reply:
Ugh. This (and several following warnings) are barfing on a symlink. There should be no newline at such files.